pkg/types/openstack/platform.go

```diff
@@ -22,7 +22,10 @@ type Platform struct {
 	ExternalNetwork string `json:"externalNetwork,omitempty"`
 
 	// FlavorName is the name of the compute flavor to use for instances in this cluster.
-	FlavorName string `json:"computeFlavor"`
+	// Deprecated: this value is not used by the installer, use type in the defaultMachinePlatform section
+	// to define the flavor.
```

Comment:
"type" or "flavorName", in func "convertOpenStack" it reads 
config.Platform.OpenStack.DefaultMachinePlatform.FlavorName = config.Platform.OpenStack.FlavorName

Reply:
done